libraries/AP_OpticalFlow/AP_OpticalFlow.h

```diff
@@ -135,6 +138,7 @@ class AP_OpticalFlow
     AP_Int16 _yawAngle_cd;          // yaw angle of sensor X axis with respect to vehicle X axis - centi degrees
     AP_Vector3f _pos_offset;        // position offset of the flow sensor in the body frame
     AP_Int8  _address;              // address on the bus (allows selecting between 8 possible I2C addresses for px4flow)
+    AP_Float  _height;              // height of the sensor above the ground. Only used in rover
```

Comment:
Let's rename this to _height_override as well.

Reply:
Fixed